README.md

```diff
@@ -150,6 +152,20 @@ a playbook and executed before the test run, or a single shell command which
 will be executed using the Ansible `raw` module (so the two examples above are
 exactly equivalent).
 
+Use the `min_ansible_version` key to indicate the minimum version of
+ansible that can be used.  For example, if you cannot use ansible 2.7
+to manage Fedora 32 hosts, use something like this:
+```json
+    {
+      "name": "fedora-32",
+      "source": "http://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2",
+      "upload_results": false,
+      "min_ansible_version": "2.8"
+    },
+```
+This means the CI environment will not run tests using ansible 2.7 to
```

Comment:
```suggestion
This means the CI environment will not run tests using ansible 2.7 or earlier to
```

Reply:
fixed in https://github.com/linux-system-roles/test-harness/pull/87/commits/e08a9eff0dd3e7052bcbad465dfdc921caf1464f